torch/_dynamo/variables/tensor.py

```diff
@@ -273,14 +283,17 @@ def dynamic_getattr(self, tx: "InstructionTranslator", name):
             raise NotImplementedError
 
         real_value = getattr(_input_associated_real_value, name)
-        if callable(real_value):
-            # Callables have more nuanced handling, and we should let the existing system delegate here.
-            # Raising was past behavior and so should always be sound to fall back.
-            # Note - at a certain point we may want to handle
-            raise NotImplementedError
 
         attr_source = AttrSource(self.source, name)
         install_guard(attr_source.make_guard(GuardBuilder.HASATTR))
+
+        # Typically we'd want to use variable builder here
+        # but unfortunately id(real_value.__self__) is not id(<original value>)
+        if is_bound_tensor_method(real_value):
+            from .misc import GetAttrVariable
+
+            return GetAttrVariable(self, name, source=attr_source)
```

Comment:
The reason we care is that if we build using VariableBuilder we introduce guards on `__self__`, which in some cases will have a different `id` than the value the attribute was extracted from. We then hit an error during alias detection if in subsequent calls the `id` is the same. The outcome that I really want in `x.sin()` for example is that we guard on x, don't guard on `__self__` because `__self__` is understood to be the same as `x` regardless of the `id`. Note that I only do this for base tensor methods which we control, in other cases (like below) we will fall through to builder.